sdk/modelsrepository/Azure.IoT.ModelsRepository/src/ModelsRepositoryClientOptions.cs

```diff
@@ -32,17 +33,34 @@ public enum ServiceVersion
         /// </summary>
         public ServiceVersion Version { get; }
 
+        /// <summary>
+        /// Gets the <see cref="TimeSpan"/> for which the client considers repository metadata stale.
+        /// </summary>
+        public TimeSpan MetadataExpiry { get; }
+
+        /// <summary>
+        /// Gets the default <see cref="TimeSpan"/> for <see cref="MetadataExpiry"/>.
+        /// </summary>
+        public static TimeSpan DefaultMetadataExpiry { get { return TimeSpan.FromMinutes(10); } }
```

Comment:
Alternatively, you can have a private constant here and take care of it in the constructor.

Reply:
I would have preferred this route but unfortunately the default parameter value needs to be a compile-time constant and in C#/.NET the TimeSpan type can't be declared const (one special case is to use `default` like `TimeSpan metadataExpiry = default`, which is not optimal in this context).

If the `DefaultMetadataExpiry` route, even making it private, is not preferable then one alternative could be to expose another type, like an integer (i.e. `int metadataExpiryMinutes` ) that can support default assignment from a private constant value.